js/dialog.js

```diff
@@ -28,9 +30,28 @@ export function updateDetailsDialog(data, cardUrl) {
     <p>Artist: ${data["Artist Credit"]} | Writer: ${data["Writer Credit"]}</p>
   `
   );
-  // Card
-  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].src =
-    cardUrl;
+  // Card art is applied
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].style.backgroundImage = 'url("' +
+    cardUrl + '")';
+
+  // Add animation start
+  DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0].classList.add(setCardRarity(data["Rarity Folder"]), "animated");
+  
+  // Animation reset
+  DETAILS_DIALOG_EL.getElementsByClassName("card-details-dialog-close")[0].addEventListener("click", resetCardEffects);
+  document.getElementsByClassName("dialog-overlay")[0].addEventListener("click", resetCardEffects);
+
+  // animation events
+  let dialogCard = DETAILS_DIALOG_EL.getElementsByClassName("details-dialog-card")[0];
+  dialogCard.addEventListener("animationend", function() {
+    dialogCard.classList.remove("animated");
+    if(first_open){
+      dialogCard.addEventListener("mouseover", animateCard);
+      dialogCard.addEventListener("click", animateCard);
+      first_open = false;
+    }
```

Comment:
could this be in `setupDetailsDialog` so that the global variable isn't needed?

Reply:
Thanks, it didn't actually make a whole lot of sense until I took a closer look at what was happening there. Makes sense to include it there, and the CSS bon suggested really helps with this setup.